pkg/validations/extendedversion.go

```diff
@@ -31,3 +51,60 @@ func validateBundleSignature(bundle *v1alpha1.Bundles) error {
 	}
 	return nil
 }
+
+func isExtendedSupport(clusterSpec *anywherev1.Cluster, bundle *v1alpha1.Bundles) (bool, error) {
+	versionsBundle, err := cluster.GetVersionsBundle(clusterSpec.Spec.KubernetesVersion, bundle)
+	if err != nil {
+		return false, fmt.Errorf("getting versions bundle for %s kubernetes version: %w", clusterSpec.Spec.KubernetesVersion, err)
+	}
+
+	endOfStandardSupport, err := parseDate("2006-01-02", versionsBundle.EndOfStandardSupport)
+	if err != nil {
+		return false, fmt.Errorf("parsing EndOfStandardSupport field format: %w", err)
+	}
+
+	return isPastDateThanToday(endOfStandardSupport), nil
+}
+
+func getLicense(clusterSpec *anywherev1.Cluster) (*jwt.Token, error) {
```

Comment:
this function only uses `clusterSpec.Spec.LicenseToken` from clusterSpec I recommend not taking in the whole `anywherev1.Cluster` object just for the kubernetes version field. I recommend just taking in what this function needs, so just the data type of `clusterSpec.Spec.LicenseToken`.